packages/cli/src/services/community-packages.service.ts

```diff
@@ -483,4 +483,18 @@ export class CommunityPackagesService {
 		const packageDirectory = this.resolvePackageDirectory(packageName);
 		await asyncExec(`rm -rf ${packageDirectory}`);
```

Comment:
Unrelated to this PR this line won't work on Windows so would be great to also fix this

Reply:
🤦🏽 sorry, I should have used `rmdir` from `fs/promises`. 
I'll create a separate PR to fix this.